Assets/MixedRealityToolkit.Examples/Experimental/HandMenuLayout/Prefabs/HandMenu_Slider_Assets/Scripts/UpdateTrackScale.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.UI;
+
+
+public class UpdateTrackScale : MonoBehaviour
```

Comment:
This class needs to be placed into a namespace (any namespace, but ideally obviously a namespace that makes sense :))

Otherwise I think that this will trip the build system (i.e. some of our packaging parts makes sure that each file has a namespace)

Reply:
I see! Would you be able to help me with that? I'm not too familiar with adding namespaces.

Reply:
Ah, I had to look way too long for this - this is a bit interesting because the hand menu solver stuff (which is the existing code) lives in a namespace that this is on (i.e. solvers) that this code right here SHOULDN'T be in (because it's not a solver).

I'm thinking maybe something like:

Microsoft.MixedReality.Toolkit.Examples.Experimental.HandMenu

Reply:
Thanks! I wrapped the script in Microsoft.MixedReality.Toolkit.Examples.Experimental.HandMenu namespace.

Reply:
Hi, @wiwei this is a pretty tiny nit but I think the guidance was to do more like `Microsoft.MixedReality.Toolkit.Experimental.Examples.HandMenu`. See [this page](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/ExperimentalFeatures.html#experimental-code-should-be-in-a-special-namespace) for guidance on namespaces for experimental features.